pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.smartbear.soapui</groupId>
+    <groupId>com.smartbear.soapui.se-fork</groupId>
```

Comment:
what does se-fork mean?

Reply:
Sorry, the pom files were commited accidently and have now been reverted.